src/conf_mode/container.py

```diff
@@ -363,9 +363,14 @@ def generate_run_arguments(name, container_config):
     if 'allow_host_pid' in container_config:
       host_pid = '--pid host'
 
+    name_server = ''
+    if 'name_server' in container_config:
+        for ns in container_config['name_server']:
+            name_server += f'--dns {ns}'
```

Comment:
Can we add 2 name-servers?

Reply:
yes, you can specify multiple name servers